kili/cli.py

```diff
@@ -21,17 +21,26 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 api_key_option = click.option(
-    '--api-key', type=str, envvar='KILI_API_KEY', required=True,
+    '--api-key', type=str, default=None,
     help=(
-        'Your Kili API key (overrides the KILI_API_KEY environment variable). '
-        'If not passed, requires the KILI_API_KEY environment variable to be set.'
+        'Your Kili API key (overrides the "KILI_API_KEY" environment variable). '
+        'If not passed, requires the "KILI_API_KEY" environment variable to be set.'
+    ),
+    show_default=(
+        'current user ("KILI_API_KEY" environment variable)'
```

Comment:
rm 'current user ('

Reply:
can you explain in a comment why `envvar` should not be used? Not completely sure to understand.

Reply:
simply because it is already handled by Kili_Client. We could keep it I guess...